packages/webapp/src/elections/components/ongoing-election-components/completed-round-segment.tsx

```diff
@@ -0,0 +1,72 @@
+import { useQuery } from "react-query";
+import { RiVideoUploadLine } from "react-icons/ri";
+
+import { queryMembers } from "_app";
+import { Button, Container, Expander } from "_app/ui";
+import { ElectionParticipantChip } from "elections";
+import { MembersGrid } from "members";
+
+import RoundHeader from "./round-header";
+
+interface CompletedRoundSegmentProps {
+    round: number;
+}
+
+export const CompletedRoundSegment = ({
+    round,
+}: CompletedRoundSegmentProps) => {
+    // TODO: The number of completed rounds is generated based on fixture data, but the contents are still mocked. Fill in contents!
+    const { data: participants } = useQuery({
+        ...queryMembers(1, 5),
+        staleTime: Infinity,
+    });
+
+    if (!participants) return <></>;
+
+    const winner = participants[2];
```

Comment:
I'd suggest we annotate what this `2` is (even if it's only temporary)

Reply:
I'll tag it with a TODO.